ui/css/treeherder-details-panel.css

```diff
@@ -417,3 +417,18 @@ div.similar-jobs .similar-job-list table tr {
 div.performance-panel {
   display: block !important;
 }
+
+.performance-panel-actions {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
I think you can use bootstrap utility classes for this. Likely even for the margin spacing below.